packages/cloudflare/src/cli/args.ts

```diff
@@ -25,32 +26,33 @@ export type Arguments = (
     }
 ) & { outputDir?: string };
 
+const config = {
+  allowPositionals: true,
+  strict: false,
+  options: {
+    skipBuild: { type: "boolean", short: "s", default: false },
+    output: { type: "string", short: "o" },
+    noMinify: { type: "boolean", default: false },
+    skipWranglerConfigCheck: { type: "boolean", default: false },
+    cacheChunkSize: { type: "string" },
+  },
+} as const satisfies ParseArgsConfig;
+
 export function getArgs(): Arguments {
-  const { positionals, values } = parseArgs({
-    options: {
-      skipBuild: { type: "boolean", short: "s", default: false },
-      output: { type: "string", short: "o" },
-      noMinify: { type: "boolean", default: false },
-      skipWranglerConfigCheck: { type: "boolean", default: false },
-      cacheChunkSize: { type: "string" },
-    },
-    allowPositionals: true,
-  });
+  const { positionals, values } = parseArgs(config);
 
-  const outputDir = values.output ? resolve(values.output) : undefined;
+  const outputDir = typeof values.output === "string" ? resolve(values.output) : undefined;
   if (outputDir) assertDirArg(outputDir, "output", true);
 
-  const passthroughArgs = getPassthroughArgs();
-
   switch (positionals[0]) {
     case "build":
       return {
         command: "build",
         outputDir,
         skipNextBuild:
-          values.skipBuild || ["1", "true", "yes"].includes(String(process.env.SKIP_NEXT_APP_BUILD)),
+          !!values.skipBuild || ["1", "true", "yes"].includes(String(process.env.SKIP_NEXT_APP_BUILD)),
```

Comment:
`!!` should not be needed as we have default value ?

Reply:
Disabling strict mode to stop failures on invalid params changes the return type to be less strict